pipeline/scraper/explorer.js

```diff
@@ -36,6 +48,27 @@ function writeScrapedWords(word, location) {
     });
 }
 
+
+/**
+ * Used returns an array where each line is a search term.
+ * @param {*the location of the file that is to be read} file_location 
+ */
+function openSearchTerms(file_location) {
```

Comment:
not in use right now, but if we wanted to add external files of search terms. really they should be in a separate file as exports i guess!